quipucords/fingerprinter/task.py

```diff
@@ -141,41 +149,43 @@ def format_products(redhat_products, is_rhel):
         return products
 
     @staticmethod
-    def format_system_profile(host):
-        """Grab facts from original host for system profile.
+    def format_system_profile(fingerprint_dict):
+        """Grab facts from original fingerprint_dict for system profile.
 
-        :param host: <dict> the host to pull facts from
+        :param fingerprint_dict: <dict> the fingerprint_dict to pull facts from
         :returns: a list with the system profile facts.
         """
         qpc_to_system_profile = {
             'infrastructure_type': 'infrastructure_type',
-            'architecture': 'architecture',
+            'architecture': 'arch',
```

Comment:
Is this really supposed to be `arch`? In my previous PR, I changed it from `arch` to `architecture` after a talk with Cody.

Reply:
Yes.  See https://github.com/RedHatInsights/insights-host-inventory/blob/system_profile_changes/swagger/api.spec.yaml#L798

@abaiken pointed this mistake out to me before she took off.